R/support_panel.R

```diff
@@ -14,9 +14,21 @@
 #' @param alt_href Alternative link to the parent publication (if not hosted on
 #' Explore Education Statistics)
 #' @param form_url URL for a feedback form for the dashboard
+#' @param feedback_custom_text A single vector or a combined vector wrapped in
```

Comment:
Not sure we need to add the option to customise the feedback section? Can't see an ask for it in #54?